Library/Formula/wartremover.rb

```diff
@@ -0,0 +1,31 @@
+class Wartremover < Formula
+  desc "Flexible Scala code linting tool"
+  homepage "https://github.com/puffnfresh/wartremover"
+  url "https://github.com/puffnfresh/wartremover/archive/v0.13.tar.gz"
+  sha256 "65d2f9907662bc2cc892801055713dd80a7fc25ade5091cb534bfc51eaae626b"
+
+  head "https://github.com/puffnfresh/wartremover.git", :branch => "master"
```

Comment:
Should be [master](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/download_strategy.rb#L549-L554).